### PR TITLE
Add AlMashhad

### PR DESCRIPTION
**Pull request by UltraHDR**

### CODE REVIEW

**Review by freearhey — CHANGES_REQUESTED** [review]
Please update your request:

- the logo link is geoblocked

**Comment by UltraHDR** [comment]
> Please update your request:
> 
>     * the logo link is geoblocked

Sorry, I don't have an alternative link and imgur upload is not working

File: data/channels.csv [hunk 1]
```diff
@@ -893,6 +893,7 @@ AlmaLusa.pt,Alma Lusa,,,,PT,,,c/PT,por,,FALSE,,,,,https://i.imgur.com/LnLFf8D.pn
 AlMamlakaTV.jo,Al Mamlaka TV,,,,JO,,,c/JO,ara,general,FALSE,,,,https://www.almamlakatv.com/,https://i.imgur.com/6njRt6c.png
 AlManar.lb,Al-Manar,,,,LB,,,c/LB,ara,news,FALSE,1991-06-04,,,https://www.almanar.com.lb/,https://i.imgur.com/sOQLV7L.png
 AlMasarAlOulaSatelliteChannel.iq,Al-Masar Al-Oula Satellite Channel,,,,IQ,,,c/IQ,ara,,FALSE,,,,,https://i.imgur.com/Mxao7k6.png
+AlMashhad.ae,Al Mashhad,,,,AE,,,c/AE,ara,,FALSE,,,,https://almashhad.com,https://almashhad.com/images/apple-touch-icon.png
```
Comment (freearhey) [inline data/channels.csv:896]:
```suggestion
AlMashhad.ae,Al Mashhad,,,,AE,,,c/AE,ara,,FALSE,,,,https://almashhad.com,https://i.imgur.com/6G6fxs5.png
```

Comment (UltraHDR) [reply data/channels.csv:896]: @freearhey Thankyou

**Review by freearhey — APPROVED** [review]

**Pull request merged**